.github/workflows/release.yml

```diff
@@ -23,11 +23,11 @@ jobs:
       - name: Build NPM Package
         run: deno task dnt --version ${{steps.get_tag_version.outputs.tag_version}} --server ${{steps.get_tag_version.outputs.tag_version}}
       - run: npm pack
```

Comment:
Now that the dnt task runs pack, this step shouldn't be necessary.

Reply:
does this look right

```patch
diff --git a/.github/workflows/release.yml b/.github/workflows/release.yml
index 15ec856..aca45c6 100644
--- a/.github/workflows/release.yml
+++ b/.github/workflows/release.yml
@@ -22,12 +22,10 @@ jobs:
           registry-url: "https://registry.npmjs.org"
       - name: Build NPM Package
         run: deno task dnt --version ${{steps.get_tag_version.outputs.tag_version}} --server ${{steps.get_tag_version.outputs.tag_version}}
-      - run: npm pack
-        working-directory: "./target/npm/capi"
       - uses: actions/upload-artifact@v3
         with:
           name: package
-          path: "./target/npm/capi/*.tgz"
+          path: "./target/npm/artifacts/capi*.tgz"
       - uses: octokit/request-action@v2.x
         if: startsWith(github.ref, 'refs/tags/')
         with:
```

Reply:
Sure